modules/core/api/farm_api.api.php

```diff
@@ -28,6 +28,32 @@ function hook_farm_api_meta_alter(array &$data) {
   $data['mykey'] = 'myvalue';
 }
 
+/**
+ * Allow entity types to be included in JSON:API resources.
+ *
+ * @return string[]
+ *   Returns an array of entity type IDs.
+ */
+function hook_farm_api_allow_resource_types() {
+
+  // Allow block and view entities.
+  return ['block', 'view'];
+}
+
+/**
+ * Alter allowed entity types to be included in JSON:API resources.
+ *
+ * @param string[] $entity_types
+ *   An array of entity type IDs allowed by other modules.
+ */
+function hook_farm_api_allow_resource_types_alter(&$entity_types) {
+
+  // Disable view entities.
+  if (in_array('view', $entity_types)) {
+    unset($entity_types[array_search('view', $entity_types)]);
+  }
+}
```

Comment:
* Why allow the 'view' entity type only to later remove it again?
* Maybe the function docs can say the "why" not the "what" - roughly what the function does is trivial (and in the function's name), but why we want to do that (and sometimes how it works) is less clear.

Reply:
ah, right! Makes sense.